Sources/PaymentButtons/PayPalButton.swift

```diff
@@ -125,13 +122,21 @@ public extension PayPalButton {
 struct PayPalButtonView: View {
 
     var body: some View {
+
         PayPalButton.Representable()
     }
 }
 
 struct PayPalButtonView_Preview: PreviewProvider {
 
     static var previews: some View {
-        PayPalButtonView()
+        PayPalButtonWrapper()
```

Comment:
Can we use `PayPalButtonView` here instead for the preview? Then we can remove the wrapper that looks like a duplicate of the above.

```suggestion
        PayPalButtonView()
```

Reply:
That's a good point, will make the change